packages/astro/components/viewtransitions.css

```diff
@@ -43,6 +43,36 @@
 	}
 }
 
+@layer astro {
+	::view-transition-old(root) {
+		animation-name: astroFadeOut;
+		animation-duration: 180ms;
+		animation-timing-function: cubic-bezier(0.76, 0, 0.24, 1);
+		animation-fill-mode: both;
+	}
+	
+	::view-transition-new(root) {
+		animation-name: astroFadeIn;
+		animation-duration: 180ms;
+		animation-timing-function: cubic-bezier(0.76, 0, 0.24, 1);
+		animation-fill-mode: both;
+	}
+}
+
+:root[data-astro-transition-fallback="old"]:not([data-astro-transition-scope]) {
```

Comment:
need to add comments, but this says "fallback on the root when there is no transition: directive used"